dotcom-rendering/src/components/BylineLink.tsx

```diff
@@ -141,6 +146,41 @@ export const BylineLink = ({
 		);
 	});
 
+	if (
+		(design === ArticleDesign.Video || design === ArticleDesign.Audio) &&
+		source
```

Comment:
Should this be `isString(source)` as `source` could also be an empty string which is falsey?

Reply:
Sometimes it comes through as an empty string, in which case we don't want to render the `Source: ` bit either. `isString(source) && source !== ''` would be clearer though!